src/Index.html

```diff
@@ -18,37 +17,22 @@ <h4 class="card-title">Log in to Mental Health Website</h4>
             <p class="card-text">Login Page</p>
         </div>
     </div>
-
 </div>
-
 <br>
 <div class="form-group" style="display:block; justify-items: center; align-items: center; margin: auto; max-width: 30%">
-
     <div class="form-group" >
         <label for="usernameText">User Name or email address:</label>
-        <input type="text"
-               class="form-control" name="frmUserName" id="usernameText" aria-describedby="helpId"
-               placeholder="User Name" required>
-
+        <input type="text" class="form-control" name="frmUserName" id="usernameText" aria-describedby="helpId" placeholder="User Name" required>
     </div>
-
     <div class="form-group">
         <label for="passwordText">Password:</label>
-        <input type="password" class="form-control" name="frmPassword" id="passwordText" placeholder="Password"
-               required>
+        <input type="password" class="form-control" name="frmPassword" id="passwordText" placeholder="Password" required>
     </div>
-
     <div class="text-center">
         <button type="submit" class="btn btn-primary">Log In</button>
-
-
-        <input name="btnSignUp" id="SignUp" class="btn btn-primary" type="button" value="Sign Up">
+    <input name="btnSignUp" id="SignUp" class="btn btn-primary" type="button" value="Sign Up">
     </div>
-
 </div>
-
-
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure